actioncable/Rakefile

```diff
@@ -1,15 +1,14 @@
 require 'rake/testtask'
 require 'pathname'
-require 'sprockets'
-require 'coffee-script'
 require 'action_cable'
+require 'blade'
 
 dir = File.dirname(__FILE__)
 
 task :default => :test
 
 task :package => "assets:compile"
-task "package:clean" => "assets:clean"
+task "package:clean"
```

Comment:
It was indeed the only one, because it's the only component that currently produces a temporary file to clean up. I don't see how a change in the mechanism we use to generate the file obviates such a task... but I agree there's no point leaving it here empty.